Mage.Client/src/test/java/mage/client/game/MultiConnectTest.java

```diff
@@ -70,25 +90,218 @@ public void connected(String message) {
             connected++;
         }
 
-        @Override
         public void disconnected(boolean askToReconnect) {
             logger.info("disconnected");
         }
 
-        @Override
         public void showMessage(String message) {
             logger.info("showMessage: " + message);
         }
 
-        @Override
         public void showError(String message) {
             logger.info("showError: " + message);
         }
 
         @Override
-        public void processCallback(ClientCallback callback) {
-            logger.info("processCallback");
+        public void inform(String title, String message, MessageType type) {
+            throw new UnsupportedOperationException("Not supported yet."); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody
+        }
```

Comment:
Same as the other class. All of these these to be either left empty (with a comment explaining that they should be empty) or they need to be implemented.